src/AbcSmc.cpp

```diff
@@ -715,8 +849,8 @@ bool AbcSmc::build_database(const gsl_rng* RNG) {
         ss << "insert into " << PAR_TABLE << " values ( " << rowid << ", '" << seed << "'"; for (size_t j = 0; j < npar(); j++)  ss << ", " << parrow[j]; ss << " );";
         _db_execute_stringstream(db, ss);
 
-        if (use_transformed_pars) {
-            vector<double> upars = do_complicated_untransformations(_model_pars, parrow);
+        if (_par_modification_map.size()) {
```

Comment:
blarg.  some other way to make these a bit more readable, rather than commenting all of them?  the flag was easier to make sense of regarding intent

Reply:
perhaps. propose: issue it for now, and i'll get rid of this problem when i do the DB revisions.